src/store/defaultIndicators.json

```diff
@@ -303,5 +303,22 @@
+  },
+  "macd": {
+    "name": "MACD",
+    "version": "1.0.0",
```

Comment:
sadly theres no versionning for indicator. it's app version at which the indicator can be added to the existing clients
so anything above 3.4.0 (current version)

Reply:
Ok, thanks.